tests/e2e/e2e.go

```diff
@@ -119,3 +119,37 @@ func waitForDeleteCmd(cmd string, object string) bool {
 		}
 	}
 }
+
+// waitForServiceCreateCmd runs a command until it finds
+// the deleted resource to be absent in the list of relevant objects from cmd executed
+// It accepts 3 arguments, cmd (command to be run)
+// object (the created object).
+// & status (the required status of the service)
+// It times out if the command doesn't fetch the
+// expected output  within the timeout period (5m).
+func waitForServiceCreateCmd(cmd string, object string, status string) bool {
```

Comment:
This is 3rd function that is almost the same ([waitForCmdOut(cmd string, expOut string)](https://sourcegraph.com/github.com/redhat-developer/odo/-/blob/tests/e2e/e2e.go?utm_source=inline-symbol#L71:6-71:19) [waitForDeleteCmd(cmd string, object string)](https://sourcegraph.com/github.com/redhat-developer/odo/-/blob/tests/e2e/e2e.go?utm_source=inline-symbol#L100:6-100:22) and this one). 

If we can't create one, function that will cover all 3 use cases that it would be good to at least abstract away the common parts like timeout handling and command execution part